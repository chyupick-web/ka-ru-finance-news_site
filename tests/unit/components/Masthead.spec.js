import { shallowMount, createLocalVue } from '@vue/test-utils'
import Component from '@/components/Masthead.vue'
import BootstrapVue from 'bootstrap-vue'

const localVue = createLocalVue()
localVue.use(BootstrapVue)

describe('Masthead.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Component, { localVue })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
