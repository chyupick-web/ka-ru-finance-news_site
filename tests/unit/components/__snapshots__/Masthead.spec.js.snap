// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Masthead.vue matches snapshot 1`] = `
<div class="mt-2 mb-2 container">
  <div class="row">
    <div class="col">
      <h2 class="border-bottom border-sei-gray-lighter mt-2 mb-4 pb-3 display-4"><a href="https://sei.cmu.edu">Software Engineering Institute</a></h2>
      <h3 class="mb-3">SEI on GitHub</h3>
      <div class="row">
        <div class="col-lg-8">
          <p>Use this site to access and download the software, tools, and methods that the SEI creates, tests, refines, and disseminates. Organizations and individuals worldwide use these technologies and management techniques to improve the results of software projects, the quality and behavior of software systems, and the security and survivability of networked systems.</p>
          <p>For more information about the SEI and its other work, visit <a href="https://sei.cmu.edu" class="text-cmu-red">sei.cmu.edu.</a></p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
